addons/account/models/account_tax.py

```diff
@@ -341,6 +342,10 @@ def _sanitize_vals(self, vals):
 
     @api.model_create_multi
     def create(self, vals_list):
+        for vals in vals_list:
+            if vals.get('repartition_line_ids') and vals.get("invoice_repartition_line_ids") and vals.get("refund_repartition_line_ids"):
+                vals.pop('repartition_line_ids')
```

Comment:
Can be managed in `_sanitize_vals` to keep all the logic in one place
I would also remove the `invoice_repartition_line_ids` and the `refund_repartition_line_ids` instead as those are not the stored field, but `repartition_line_ids` is

Reply:
Hello @william-andre ,
I have done the requested changes in '_sanitize_vals' with similar logic. Please check them. Also popping 'invoice_repartition_line_ids' and ''refund_repartition_line_ids'' may break the usual flow. If there are any changes to be done please do mention about them below.
Thank you